docs/operational_guide/multiple_m3db_clusters.md

```diff
@@ -0,0 +1,125 @@
+## Write to multiple M3DB clusters via M3Coordinator
+
+### Overview:
+
+Default M3 architecture has the M3Coordinator writing to and aggregating meterics from a single M3DB cluster. If wanting to add more than one, follow the below instructions. 
+
+Use case(s):
+- Sending metrics to different namespaces for different retention periods, etc.
+
+### Instructions: 
+
+**Note:** Adding mutliple M3DB clusters to m3coordinator using the m3aggregator requires clusterManagement
+
+**Note:** When making API requests, an environment header needs to be set. 
+
+1. Add clusterManagement to congfig file:
+
+Example of clusterManagement config:
+
+```bash
+clusterManagement:
+  etcd:
+    env: default_env
+    zone: embedded
+    service: m3db
+    cacheDir: /data/m3kv_default
+    etcdClusters:
+      - zone: embedded
+        endpoints:
+          - 10.33.131.173:2379
```

Comment:
put in placeholder etcd ip addresses. e.g. `<ETCD_IP_1>` or something similar (can copy from other config files?)